spec/formtastic_integration_spec.rb

```diff
@@ -0,0 +1,49 @@
+# rubocop:disable AlignArray
+require 'spec_helper'
+require 'formtastic'
+require 'formtastic/inputs/trix_editor_input'
+
+describe Trix do
+  include ActionPack
+  include ActionView::Context if defined?(ActionView::Context)
+  include ActionController::RecordIdentifier if defined?(ActionController::RecordIdentifier)
+  include ActionView::Helpers::FormHelper
+  include Formtastic::Helpers::FormHelper
+  include ActionDispatch::Routing::PolymorphicRoutes
+  include ActionView::RecordIdentifier if defined?(ActionView::RecordIdentifier)
+  Formtastic::FormBuilder.input_class_finder = Formtastic::InputClassFinder
+
+  class Application < Rails::Application
+    # Configure the default encoding used in templates for Ruby 1.9.
+    config.encoding = 'utf-8'
```

Comment:
If you mean the config variables inside the fake Application class, the test will still work without them, but I'm not sure that won't cause any problem in the future.
Regarding the include part, unfortunately that is the bare minimum setup, otherwise the test will complain like `undefined method 'form_for'` or `undefined method 'post_path'` etc...